src/server/api/lib/twilio.js

```diff
@@ -152,6 +153,11 @@ function parseMessageText(message) {
   return params;
 }
 
+async function getMessagingServiceSid(contact) {
+  const campaign = await Campaign.get(contact.campaign_id);
```

Comment:
get the campaign from the cache and fall back to organization/global messaging service if it's null